src/styles.css

```diff
@@ -483,7 +505,7 @@ nav .burger-icon.burger-open {
 }
 .temp-label-x,
 .temp-label-y {
-  font-size: 0.75rem;
+  font-size: 14pt;
```

Comment:
why not variable? why not rem units?

Reply:
Because this is for temperature SVG and it is being scaled on its own.